lib/github_fetcher.rb

```diff
@@ -1,10 +1,10 @@
-require "octokit"
+require 'octokit'
 
 class GithubFetcher
   def initialize(team)
-    @organisation = ENV["SEAL_ORGANISATION"]
-    @github = Octokit::Client.new(access_token: ENV["GITHUB_TOKEN"])
-    @github.api_endpoint = ENV["GITHUB_API_ENDPOINT"] if ENV["GITHUB_API_ENDPOINT"]
+    @organisation = ENV['SEAL_ORGANISATION']
```

Comment:
Just a thing I noticed (haven't reviewed the rest of the PR, only just noticed this in passing): the changes here don't match with the rules configured in `rubocop-govuk` (e.g. we prefer double quotes over single quotes).  I think you're missing the `.rubocop.yml` configuration file.

Reply:
Thanks, I'll change that!

Reply:
I've added the govuk config and re-run Rubocop now